pkg/internal/apis/config/validate_test.go

```diff
@@ -432,45 +432,66 @@ func TestPortValidate(t *testing.T) {
 }
 
 func TestValidatePortMappings(t *testing.T) {
-	newPortMapping := func(addr string, port int, protocol string) PortMapping {
+	newPortMapping := func(addr string, hostPort, containerPort int, protocol string) PortMapping {
 		return PortMapping{
-			HostPort:      int32(port),
+			HostPort:      int32(hostPort),
+			ContainerPort: int32(containerPort),
 			ListenAddress: addr,
 			Protocol:      PortMappingProtocol(protocol),
 		}
 	}
-	errMsg := "port mapping with same listen address, port and protocol already configured"
+	errMsg := "port mapping with same listen address, host port and protocol already configured"
 	cases := []struct {
 		testName     string
 		portMappings []PortMapping
 		expectErr    string
 	}{
+		{
+			testName: "unique container ports",
+			portMappings: []PortMapping{
+				newPortMapping("", 0, 1000, ""),
+				newPortMapping("", 0, 2000, ""),
+				newPortMapping("", 0, 3000, ""),
+				newPortMapping("", 0, 4000, ""),
+			},
+			expectErr: "",
+		},
+		{
+			testName: "duplicate container ports",
+			portMappings: []PortMapping{
+				newPortMapping("", 0, 1000, ""),
+				newPortMapping("", 0, 2000, ""),
+				newPortMapping("", 0, 3000, ""),
+				newPortMapping("", 0, 3000, ""),
+			},
+			expectErr: "",
```

Comment:
both testcases are the same, aren't they?

Reply:
the second one has a duplicate containerPort, which is allowed by both kind and docker.

Reply:
oh, heh, I started at it for one minute and didn't notice it XD